src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -62,7 +62,7 @@ protected internal bool IsStandardContract(UInt160 hash)
 
             // Try to find it in the transaction
 
-            if (ScriptContainer is Transaction tx)
+            if (ScriptContainer is Transaction tx && tx.Witnesses is not null)
```

Comment:
Will this work for the case of `invokefunction` RPC method calls?
I.e., if I call a contract method with `invokefunction` and the method contains a call to `System.Contract.isStandard`.

Reply:
Yes, it should return false if there are no witness